<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TurboPage</class>
 <widget class="QWidget" name="TurboPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turbo Stake</string>
  </property>


  <!-- horizLayout -->
  <layout class="QHBoxLayout" name="horizLayout">
   <property name="bottomMargin">
    <number>8</number>
   </property>

   <!-- vertLayoutA -->
   <item>
    <layout class="QVBoxLayout" name="vertLayoutA">
     <property name="spacing">
      <number>4</number>
     </property>

     <!-- tableFrame -->
     <item>
        <widget class="QFrame" name="tableFrame">
           <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>10</y>
              <width>450</width>
              <height>400</height>
             </rect>
            </property>

           <property name="fixedSize" stdset="0">
            <size>
             <width>450</width>
             <height>400</height>
            </size>
           </property>

       <!-- horizLayoutA1 -->
<!--
       <item>
 -->
        <layout class="QHBoxLayout" name="horizLayoutA1">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QcGaugeWidget" name="mainGauge">
           <property name="fixedSize" stdset="0">
            <size>
             <width>300</width>
             <height>1</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="turboTableView">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <property name="fixedSize" stdset="0">
            <size>
             <width>200</width>
             <height>1</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
<!--
       </item>
-->
       <!-- /horizLayoutA1 -->

     </widget>
    </item>
     <!-- tableFrame -->

       <!-- vertLayoutA2 -->
       <item>
        <layout class="QVBoxLayout" name="vertLayoutA2">
         <property name="spacing">
          <number>2</number>
         </property>


         <!-- horizLayoutA2a -->
         <item>
          <layout class="QHBoxLayout" name="horizLayoutA2a">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="labelTxComment">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Synergy Address: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeft|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="buddy">
              <cstring>editAddress</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValidatedLineEdit" name="editAddress">
             <property name="fixedSize" stdset="0">
              <size>
               <width>500</width>
               <height>1</height>
              </size>
             </property>
             <property name="minimumSizeHint" stdset="0">
              <size>
               <width>500</width>
               <height>1</height>
              </size>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Enter a Synergy Address</string>
             </property>
            </widget>
           </item>
           <item>
             <widget class="QPushButton" name="startButton">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
                  <string>Turbo...</string>
              </property>
             </widget>
           </item>
          </layout>
         </item>
         <!-- /horizLayoutA2a -->

         <item>
          <widget class="QCustomPlot" name="plotAddress"/>
         </item>

        </layout>
       </item>
       <!-- /vertLayoutA2 -->

    </layout>
   </item>
   <!-- /vertLayoutA -->


  </layout>
  <!-- /horizLayout -->

 </widget>

 <customwidgets>
<!--
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
-->
 </customwidgets>
 <resources/>
 <connections/>
</ui>
